### PR TITLE
metadata, duplicate headers, disallowed html

### PR DESCRIPTION
**Pull request by v-emilypr**

### CODE REVIEW

File: api-reference/v1.0/api/driveitem-get.md [hunk 1]
```diff
@@ -93,7 +93,7 @@ GET /me/drive/root
 ---
 
 
-## Response
+## Example Response
```
Comment (Lauragra) [inline api-reference/v1.0/api/driveitem-get.md:96]: Please change the fix here - this should be an H2, so it should appear like so:
### Response

File: api-reference/v1.0/api/filter-apply.md [hunk 2]
```diff
@@ -3,7 +3,7 @@ title: "Filter: apply"
 description: "Apply the given filter criteria on the given column."
 ms.localizationpriority: medium
 author: "ruoyingl"
-ms.prod: ""
+ms.prod: non-product-specific
```
Comment (Lauragra) [inline api-reference/v1.0/api/filter-apply.md:6]: It looks like the ms.prod value here should be "workbooks-and-charts".

File: api-reference/v1.0/api/filter-clear.md [hunk 3]
```diff
@@ -3,7 +3,7 @@ title: "Filter: clear"
 description: "Clear the filter on the given column."
 ms.localizationpriority: medium
 author: "ruoyingl"
-ms.prod: ""
+ms.prod: non-product-specific
```
Comment (Lauragra) [inline api-reference/v1.0/api/filter-clear.md:6]: workbooks-and-charts

File: api-reference/v1.0/api/formatprotection-get.md [hunk 4]
```diff
@@ -3,7 +3,7 @@ title: "Get FormatProtection"
 description: "Retrieve the properties and relationships of formatprotection object."
 ms.localizationpriority: medium
 author: "ruoyingl"
-ms.prod: ""
+ms.prod: non-product-specific
```
Comment (Lauragra) [inline api-reference/v1.0/api/formatprotection-get.md:6]: workbooks-and-charts

File: api-reference/v1.0/api/formatprotection-update.md [hunk 5]
```diff
@@ -3,7 +3,7 @@ title: "Update formatprotection"
 description: "Update the properties of formatprotection object."
 ms.localizationpriority: medium
 author: "ruoyingl"
-ms.prod: ""
+ms.prod: non-product-specific
```
Comment (Lauragra) [inline api-reference/v1.0/api/formatprotection-update.md:6]: workbooks-and-charts

File: api-reference/v1.0/api/icon-get.md [hunk 6]
```diff
@@ -3,7 +3,7 @@ title: "Get Icon"
 description: "Retrieve the properties and relationships of icon object."
 ms.localizationpriority: medium
 author: "ruoyingl"
-ms.prod: ""
+ms.prod: non-product-specific
```
Comment (Lauragra) [inline api-reference/v1.0/api/icon-get.md:6]: workbooks-and-charts

File: api-reference/v1.0/api/icon-update.md [hunk 7]
```diff
@@ -3,7 +3,7 @@ title: "Update icon"
 description: "Update the properties of icon object."
 ms.localizationpriority: medium
 author: "ruoyingl"
-ms.prod: ""
+ms.prod: non-product-specific
```
Comment (Lauragra) [inline api-reference/v1.0/api/icon-update.md:6]: workbooks-and-charts

File: api-reference/v1.0/api/list-create.md [hunk 8]
```diff
@@ -108,7 +108,7 @@ Content-Type: application/json
 ---
 
 
-## Response
+## Example Response
```
Comment (Lauragra) [inline api-reference/v1.0/api/list-create.md:111]: This should be:
\### Response

**Comment by Lauragra** [comment]
@Erikre , @jaiprakashmb , please advise as to whether the updates to the Intune topics look okay to you?

**Comment by v-emilypr** [comment]
@Lauragra, thank you for your guidance on this PR. I have made the changes as you advised.

**Comment by v-emilypr** [comment]
@Lauragra Happy to help if there are any further changes that should be made on this PR before its merged. Thanks!

**Pull request merged**